### PR TITLE
Add option to select acceptor name

### PR DESCRIPTION
**Pull request by simo5**
This option is useful to select and allow only a specific credential
when keys for multiple principals are available in a keytab.

### CODE REVIEW

File: src/mod_auth_gssapi.c [hunk 1]
```diff
@@ -1706,6 +1706,23 @@ static const char *mag_basic_auth_mechs(cmd_parms *parms, void *mconfig,
...
+{
+    struct mag_config *cfg = (struct mag_config *)mconfig;
+    gss_buffer_desc bufnam = { strlen(w), (void *)w };
+    uint32_t maj, min;
+
+    maj = gss_import_name(&min, &bufnam, GSS_C_NT_HOSTBASED_SERVICE,
+                          &cfg->acceptor_name);
+    if (GSS_ERROR(maj)) {
+        ap_log_error(APLOG_MARK, APLOG_WARNING, 0, parms->server,
+                     "gss_import_name([%s]) failed", w);
```
Comment (frenche) [inline src/mod_auth_gssapi.c]: Why not return "error: malformed name" here? I think it is more useful to detect the error at init time.

Comment (simo5) [inline src/mod_auth_gssapi.c]: Are you suggesting to log at APLOG_ERROR level, or also to return a non-NULL error ?

Comment (frenche) [inline src/mod_auth_gssapi.c]: Return non-null, so the directive won't get silently ignored.

**Comment by simo5** [comment]
Ok now a bad name (like an empry name) returns an error.

**Pull request merged**